install-data.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+import os
+import sys
+import subprocess
+import json
+import datetime
+from xml.dom import minidom
+
+FLATPAK_ID = os.environ.get('FLATPAK_ID')
+
+EXTRA_DESCRIPTION = [
+"""This is the Open Source build of Visual Studio Code, packaged into a Flatpak. Some features are
+different from the proprietary version: There is no telemetry nor Twitter integration, and the
+logo is a different one without copyright issue. This OSS repackaging, as well as the proprietary
+repackaging in Flathub, are not supported by Microsoft.""",
```

Comment:
What's the point of mentioning extra-data version here?

Reply:
It's copied from current OSS build as-is, so dunno.

Reply:
I'd change it to something along "This build is not supported by Microsoft. Issues should be reported at https://github.com/flathub/appid." and drop the setence with link to https://github.com/flathub/com.visualstudio.code.oss/issues/6#issuecomment-380152999.